app/helpers/url_safe_serializer.py

```diff
@@ -6,4 +6,7 @@
 
 
 def url_safe_serializer() -> URLSafeSerializer:
-    return URLSafeSerializer(current_app.secret_key, salt=cookie_session[EQ_SESSION_ID])
+    secret_key = current_app.secret_key
+    if secret_key is None:
+        raise ValueError("Flask application secret key is not set.")
+    return URLSafeSerializer(secret_key, salt=cookie_session[EQ_SESSION_ID])
```

Comment:
This shouldn't have changed. This would be too late. We should type ignore this but enfore secret key exists when loaded. So in `setup_secure_cookies` method.

Reply:
Yeah wasnt sure to type ignore will remove test as well 👍

Reply:
Resolved on another PR